docs/source/concepts/catvrs_model.rst

```diff
@@ -34,28 +34,158 @@ DefiningAlleleConstraint
 
 .. include:: ../def/cat-vrs/DefiningAlleleConstraint.rst
 
+**Examples**
+
+The following are example implementations of DefiningAlleleConstraint:
+
+.. collapse:: NM_004958.4(MTOR):c.5992_5993del (p.Met1998fs)
+
+   .. literalinclude:: ../../../schema/cat-vrs/json/example_canonicalAllele-ex1
+      :language: json
+      :lines: 83-172
+
+.. raw:: html
+
+   <div style="margin-top: 0.5em;"></div>
```

Comment:
Raw HTML repeated regularly like this is a sign we need to update our .css. This should be handled as a separate issue, created #149 to address later.